### PR TITLE
fix: add key_to_nodes to managed_ring and worker

### PR DESCRIPTION
**Pull request by MikaAK**
Just adding key_to_nodes to allow for replicas

### CODE REVIEW

File: lib/managed_ring.ex [hunk 1]
```diff
@@ -250,4 +250,17 @@ defmodule HashRing.Managed do
   def key_to_node(ring, key) do
     HashRing.Worker.key_to_node(ring, key)
   end
+
+  @doc """
+  Maps a key to a node on the hash ring.
```
Comment (bitwalker) [inline lib/managed_ring.ex:255]: *Nit:* Rephrase this to "Maps a key to a set of nodes on the hash ring"

**Review by bitwalker — CHANGES_REQUESTED** [review]
Looks good, just one small tweak and I can get this merged

File: lib/managed_ring.ex (hunk 1 above)
Comment (MikaAK) [reply lib/managed_ring.ex]: Done! Sorry for such a long delay @bitwalker I wasn't checking my github notifications 😆

**Review by bitwalker — APPROVED** [review]
